apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -568,22 +574,6 @@ function handleCloseEditor( calypsoPort ) {
 	} );
 }
 
-// The close button is generally overridden using the <MainDashboardButton> slot API
-// which was introduced in Gutenberg 8.2. In older editors we still need to override
```

Comment:
8.2 was more than a year ago -- I think it's finally safe to remove this.

Reply:
The [wbe readme](https://github.com/Automattic/wp-calypso/blob/04a3259bb6a27bfa149beed27db8f74146e85cf8/apps/wpcom-block-editor/README.md#L11) says `iframe-bridge-server` can appear on jetpack sites to facilitate communication between the iframe and the editor. That means we have no control over whether `MainDashboardButton` is available or not. They could be on an old version of WP from before `MainDashboardButton` was merged into core.

Reply:
Good point, I'll add this back.